sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -194,7 +194,7 @@ def load(*args, **kwargs) -> "AzureAppConfigurationProvider":
     try:
         provider._load_all(headers=headers)
     except Exception as e:
-        _prekill(start_time)
+        _min_uptime(start_time)
```

Comment:
Method names should describe the action being performed. How about `_delay_failure()`?